stdlib/public/core/Array.swift

```diff
@@ -1784,7 +1784,7 @@ extension Array {
 
 extension Array: _HasCustomAnyHashableRepresentation
   where Element: Hashable {
-  public func _toCustomAnyHashable() -> AnyHashable? {
+  __consuming public func _toCustomAnyHashable() -> AnyHashable? {
```

Comment:
I think I found when updating tests that process these getting spat back out that they come out as `public __consuming func` so I've done the others that way around.

Reply:
ah in fact this is just the odd-one out